vignettes/vignette.Rmd

```diff
@@ -373,24 +405,27 @@ table(fossils$formation)
 tax_check(taxdf = fossils, name = "formation", dis = 0.1)
 ```
 
-Here, we see four flagged formation name pairs, which vary in how problematic they seem. For example "Couche II" and "Couche I" make sense as an adjacent pair of formations, and likely are not a mistake. However, we also see "San Sebastián" and "San Sebastian", which should be unified under the same name, either with or without the accent included.
+Here, we see four flagged formation name pairs, which vary in how problematic they seem. For example "Couche II" and "Couche I" make sense as an adjacent pair of formations, and are likely not a mistake. However, we also see "San Sebastián" and "San Sebastian", which should be unified under the same name, either with or without the accent included.
 
 We can also look for inconsistencies in the interval names and dates. For example, an interval could be spelled two different ways, or have multiple associated dates.
 
 ```{r check_ages}
-# Again, we can use the table() function to tabulate the frequency of each interval
+# Again, we can use the table() function to tabulate the frequency of each 
+# interval
 table(fossils$early_interval)
 table(fossils$late_interval)
 
-# What are the ages associated with each interval? Any issues with the date formats?
+# What are the ages associated with each interval? Any issues with the date 
+# formats?
 fossils %>% 
-  group_by(early_interval) %>% #grouping by interval
-  distinct(max_ma.x) %>%  #pulling out the distinct values
-  arrange(early_interval) #putting the list in order so it's easier to scan for inconsistencies
+  group_by(early_interval) %>% # Grouping by interval
+  distinct(max_ma) %>%  # Pulling out the distinct values
+  arrange(early_interval) # Putting the list in order so it's easier to scan for
+                          # inconsistencies
```

Comment:
Because it is now on a new line, this comment gets put in a separate rendered chunk after the output of this pipe
![image](https://github.com/user-attachments/assets/df673086-aad6-4038-98f1-bd2f6844969d)